firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt

```diff
@@ -144,7 +144,7 @@ class FilterDialogFragment : DialogFragment() {
     }
 
     fun resetFilters() {
-        binding.let {
+        binding?.let {
```

Comment:
In this case, `binding` is never null. In order to fix the error, we might want to use `_binding` instead.
```suggestion
        _binding?.let {
```

Reply:
Crashed if i used the implementation in a fragment, calling the Dialog from a fragment. But yeah _binding?.let is better